client/rest/src/plugins/rosetta/nem/OperationParser.js

```diff
@@ -432,7 +433,8 @@ export class OperationParser {
 
 				const isExistingPurged = transactionLocation && transactionLocation.height > existingMosaicDefinitionTuple.expirationHeight;
 				if (!isExistingPurged) {
-					if (areMosaicDefinitionsEqual(mosaicDefinition, existingMosaicDefinition)) {
+					const isDescriptionSignificant = isMosaicDefinitionDescriptionSignificant(this.network, transactionLocation);
```

Comment:
If ``isDescriptionSignificant`` is true, then if the owner is the same, we want to do nothing.
From what I noticed, before the fork, the expired endpoint was triggered when the mosaic was redefined.

Reply:
so, you're saying we should always treat description as insignificant for purposes of rosetta?